test/TestUtilities/Test.Utility/CommandRunnerResult.cs

```diff
@@ -32,9 +30,8 @@ public class CommandRunnerResult
         /// </summary>
         public string AllOutput => Output + Environment.NewLine + Errors;
 
-        internal CommandRunnerResult(Process process, int exitCode, string output, string error)
+        internal CommandRunnerResult(int exitCode, string output, string error)
```

Comment:
The `Process` is in a `using` so there's no point in passing a reference to it since we only needed it's exit code.  This might make things more reliable...